SpatialGDK/Documentation/content/cross-server-rpcs.md

```diff
@@ -67,6 +67,22 @@ The tables below show where cross-server RPCs are executed based on where they w
 
 The call is not processed because this type of RPC is only for a server-worker instance to call.
 
+### Unresolved parameters in an RPC
+
+In some situations, a worker (either a client or a server) may receive an RPC with a parameter that it cannot resolve. In these instances, the GDK will output a warning. However, sometimes this behaviour is intended. To account for these cases where the behaviour is intended, you can disable the warnings on a per-RPC basis, either using C++ or using Blueprints.
+
+#### How to allow unresolved parameters in an RPC (using C++)
+
+To disable these warnings on an RPC in C++, add an `AllowUnresolvedParameters` tag to the `UFUNCTION` macro of the RPC function.
+
+```
+UFUNCTION(CrossServer, AllowUnresolvedParameters, Reliable, WithValidation)
```

Comment:
nit: I think it'd be better to keep these to a server or client rpc as they're slightly less complex